.packit.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
       distros: [RHEL-7.9-ZStream]
   identifier: tests-79to84
   tmt_plan: "^(?!.*upgrade_plugin)(?!.*c2r)(?!.*sap)(?!.*8to9)(?!.*morf)"
-  tf_post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys;curl https://copr.fedorainfracloud.org/coprs/g/oamg/leapp/repo/epel-7/group_oamg-leapp-epel-7.repo --output /etc/yum.repos.d/copr_build.repo; yum install -y leapp*master*"
+  tf_post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys; yum install -y leapp-repository*master*"
```

Comment:
Yeah, I think they implemented that already: https://github.com/packit/packit-service/issues/1544

Reply:
And here's the link for the docs: https://packit.dev/docs/testing-farm/#running-tests-with-builds-from-another-pull-request

Reply:
I am wondering if executing `/packit test <namespace>/<repo>#<pr_id>` as recommended by packit will work fine if we have leapp-repository*master* installation hardcoded as a tf_post_install_script..

Reply:
Well, to be honest, that was a super recent change they made that may not be 100% working right now. In any way, I'm hoping to test that feature here instead of my demo repository, I figured that it would be nicer to have actual (real) results instead of the made up ones I have in my repo. 

Since they included that command especially because we asked, I don't mind go to them and ask for an update on that if we see that the behavior is incorrect.